docker-compose.maintenance.yml

```
@@ -1,8 +1,6 @@
 # Todo: !!! This file seems related to our old maintenance worker for MongoDB and has to be refactored in case of using it !!!
 
-version: "3.4"
-
-services:
+ervices:
```

```suggestion
services:
```

:sob: this will be a "Request changes"